service/api_task_test.go

```diff
@@ -204,6 +204,19 @@ func TestAssignNextAvailableTaskWithDispatcherSettingsVersionLegacy(t *testing.T
 			So(h.RunningTask, ShouldEqual, "task1")
 
 		})
+		Convey("tasks with a disabled project should be removed from the queue", func() {
+			pref.Enabled = false
```

Comment:
Can this test also test pref.PatchingDisabled = true and pRef.Enabled = true?

Reply:
done